.github/workflows/ci.yaml

```
@@ -582,36 +582,202 @@ jobs:
+      - name: Install winget (required for Windows dependency installation)
+        uses: Cyberboss/install-winget@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install Windows dependencies via winget
         run: |
-          cargo test --workspace -- --format json --report-time -Z unstable-options | tee results.json
-        continue-on-error: true
+          # Verify winget is available
+          winget --version
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)